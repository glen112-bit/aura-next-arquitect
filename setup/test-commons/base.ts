import type {Config} from '@jest/types';
import path from 'path'; 

// Sync object
const config: Config.InitialOptions = {
  testEnvironment: 'jsdom',
  verbose: false,
  transform: {
    '\\.[jt]sx?$': ['babel-jest', {
      configFile: path.resolve(__dirname, '.babelrc'),
    }],
  }
};
export default config;
