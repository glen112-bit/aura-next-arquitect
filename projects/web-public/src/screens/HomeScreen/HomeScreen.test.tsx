import { HomeScreen } from "./HomeScreen";
import { customRender } from "@alura/test-commons/react-testing-library";

const render = customRender();

describe("<HomeScreen />", () => {
	it("renders the page", () => {
		const { container } = render(<HomeScreen />);

		expect(container).toMatchSnapshot();
	});
});
