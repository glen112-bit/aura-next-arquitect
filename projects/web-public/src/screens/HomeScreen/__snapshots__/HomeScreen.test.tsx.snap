// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> renders the page 1`] = `
<div>
  <main>
    <h1
      class="Text__StyledText-sc-1ul0hnu-0 cimzHl"
    >
      HomeScreen
    </h1>
    <p
      class="Text__StyledText-sc-1ul0hnu-0 cimzHl"
    >
      importando modulo local: @alura/utils/math/suma suma(2, 2): 
      4
       
    </p>
  </main>
</div>
`;
